gamecenter/api/views.py

```diff
@@ -15,6 +18,38 @@
 score_schema = ScoreSchema()
 
 
+def handle_api_key(f):
+    @functools.wraps(f)
+    def decorated_function(*args, **kwargs):
+        if "Authorization" in request.headers:
+            api_key = request.headers["Authorization"][7:]  # from the string form "Bearer <api_key>"
```

Comment:
lol, actually cmd+f+"Authorization: Bearer" found [this](http://self-issued.info/docs/draft-ietf-oauth-v2-bearer.html#authz-header), I guess it is standard procedure